### PR TITLE
Provide Windows error message

### PR DESCRIPTION
**Pull request by Froxcey**
Yes, it is already in README. I just thought it would be nice if the build system explicitly fails Windows builds.

Test package available [[here](https://github.com/Chiissu/zap/releases/tag/v0.6.0-alpha3)].

Just 2 LOC, thanks to Zig's beautiful build system.

Just wanna mention #74 

vibe

### CODE REVIEW

File: build.zig [hunk 1]
```diff
@@ -3,6 +3,8 @@ const build_facilio = @import("facil.io/build.zig").build_facilio;
 
 pub fn build(b: *std.build.Builder) !void {
     const target = b.standardTargetOptions(.{});
+    if (target.getOsTag() == .windows)
+        @panic("Facil (and Zap) doesn't support Windows, at least not currently. Try targeting Linux and using WSL\n");
```
Comment (renerocksai) [inline build.zig:7]: Please rephrase like: "Currently, Facil.io and Zap are not compatible with Windows. Consider using Linux or Windows Subsystem for Linux (WSL) instead."

**Review by renerocksai — CHANGES_REQUESTED** [review]
I want to merge this so you get the recognition.

# **AWESOME IDEA** :+1:  :+1:  :+1:  :partying_face: 

I really like it!

**Comment by Froxcey** [comment]
Sorry, I was kind of on a vacation. Anyways, it should be good now

Example

**Comment by Froxcey** [comment]
Btw I saw your talk on Showtime, and I just want to say it's awesome :D

**Comment by Froxcey** [comment]
Should `@panic` or `std.log.err` message be used? 

The difference lies in that `@panic` gives a stack trace pointing to the build script where `@panic` is invoked, whereas `std.log.err` + `std.os.exit` seems cleaner. Here are the screenshots:

`@panic`:
Version one

`std.log.err` + `std.os.exit`:
Version two

I personally prefer `std.log.err` + `std.os.exit` because the stack trace is not really adding any useful information to the already self-explanatory error message. I want the crash message to be beautiful, so I will convert this to a Draft PR temporarily before a decision is made.

**Comment by Froxcey** [comment]
I just committed the version with less chunky error message

**Review by renerocksai — APPROVED** [review]
Sorry so late but looks really good

**Pull request merged**